packages/element/src/react.js

```diff
@@ -231,6 +232,11 @@ export { lazy };
  */
 export { Suspense };
 
+/**
+ * Current version of React.
+ */
+export { version };
```

Comment:
I'd consider calling it `reactVersion` or `ReactVersion`. Mere `version` suggests that it's the version of `@wordpress/element`, which it isn't.

Reply:
Qualifying it makes sense to me.